recipe/meta.yaml

```diff
@@ -498,11 +499,11 @@ outputs:
       build:
         - {{ compiler('cxx') }}
         - {{ compiler('c') }}
-        - {{ stdlib("c") }}
+        - {{ stdlib('c') }}
         - python                              # [build_platform != target_platform]
         - cross-python_{{ target_platform }}  # [build_platform != target_platform]
         - make  # [unix]
-        - cython =0.29.36
+        - cython                              # [build_platform != target_platform]
```

Comment:
is this unpinning ok?

Reply:
`cython` is not used by OpenVINO since 2024.0
looks like we forgot to drop this dependency